component/portal/src/main/resources/db/changelog/portal-rdbms.db.changelog-1.0.0.xml

```diff
@@ -367,4 +367,9 @@
       <column name="UPDATED_DATE" type="BIGINT" defaultValueNumeric="0" />
     </addColumn>
   </changeSet>
+  <changeSet author="portal" id="1.0.0-34" dbms="mysql">
+    <sql>
+      ALTER TABLE PORTAL_SITES MODIFY COLUMN DESCRIPTION longtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
```

Comment:
Wouldn't it be enough to create database with UTF charset without putting it here in the liquibase script ?

Reply:
@ahamdi The default charset of PORTAL_SITES table is set on creation explicitly to CHARSET=UTF8 COLLATE utf8_general_ci (https://github.com/Meeds-io/gatein-portal/blob/a488e09656036b801e6490f1a264c2b11f509217/component/portal/src/main/resources/db/changelog/portal-rdbms.db.changelog-1.0.0.xml#L81), the description field will have also the same charset.
So even if we create the database with **utf8mb4 charset**, the charset of the PORTAL_SITES table is not changed and remains **utf8_general_ci**, so IMO we need to change the charset of the description column via liquibase as it is done for other tables cases.

Reply:
@azayati You need to change the charset to the correct one for the whole tables. can you change the collate in [gatein-portal/component/portal/src/main/resources/db/changelog/portal-rdbms.db.changelog-1.0.0.xml](https://github.com/Meeds-io/gatein-portal/blob/a488e09656036b801e6490f1a264c2b11f509217/component/portal/src/main/resources/db/changelog/portal-rdbms.db.changelog-1.0.0.xml#L81)

Reply:
@ahamdi you mean that we should change the charset for the whole PORTAL_SITES table ?
IMO since only the description field is considered, why we don't change only the charset of the field itself ?